packages/ui/__stories__/Button.stories.tsx

```diff
@@ -156,3 +156,53 @@ export const LinkSemantics = () => {
     </>
   )
 }
+
+export const AllVariants = () => {
```

Comment:
Can we just update the `Template` with all combinations of `variant` and `color`? Other stories are bound to the template and will update accordingly.